src/Aspire.Hosting.PostgreSQL/Aspire.Hosting.PostgreSQL.csproj

```diff
@@ -22,7 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <InternalsVisibleTo Include="Aspire.Hosting.Tests" />
+    <InternalsVisibleTo Include="Aspire.Hosting.PostgreSQL.Tests" />
```

Comment:
What else is still needed for IVT? Ideally we wouldn't need it, but since this is 1-to-1 with the project and tests, it is OK. I'd just like to understand what scenario IVT is needed for.

Reply:
PgAdminConfigWriterHook
PostgresContainerImageTags
VolumeNameGenerator